src/cgroup.rs

```diff
@@ -328,9 +351,20 @@ fn supported_controllers() -> Vec<String> {
         .collect::<Vec<String>>()
 }
 
-fn create_v2_cgroup(root: PathBuf, path: &str) -> Result<()> {
+fn supported_controllers_from_threaded_mode() -> Vec<String> {
+    let ret = String::from("cpuset cpu pids");
```

Comment:
Why not juest `vec![String::from("cpuset"), String::from("cpu"), String::from("pids")]` ?